src/store/generator.ts

```diff
@@ -158,7 +158,7 @@ const generatorStore = (set: SetState<GeneratorState>, get: GetState<GeneratorSt
     ...createGeneratorSpecificStore(set, get),
 });
 
-const { devtools } = process.env.NODE_ENV === 'development' ? require('zustand/middleware') : { devtools: undefined };
+const { devtools } = process.env.NODE_ENV === 'development' ? require('zustand/middleware') : undefined;
```

Comment:
Did I tell you to do that? If so, seems like I was wrong. This only works in the `true` case.

Reply:
But you _can_ do:

```suggestion
const { devtools } = process.env.NODE_ENV === 'development' ? require('zustand/middleware') : {};
```